superset/databases/commands/exceptions.py

```diff
@@ -131,7 +134,7 @@ class DatabaseTestConnectionDriverError(CommandInvalidError):
 
 
 class DatabaseTestConnectionUnexpectedError(CommandInvalidError):
-    message = _("Unexpected error occurred, please check your logs for details")
+    message = _(TEXT_DATABASE_CONN["database_test_connection_unexpected"])
```

Comment:
I'm not super familiar with gettext, but I think you need to move the `__()` call to `superset/config.py` where the string is defined, you can't use it with variables.

Reply:
do you know what the `_()` assuming it is for translations right?

Reply:
from flask_babel import lazy_gettext as _